app/main.py

```diff
@@ -7,13 +7,38 @@
     "The Lord of the Rings: The Return of the King",
     "Pulp Fiction",
     "The Good, the Bad and the Ugly",
-    "The Matrix",
+    "The Matrix"
 ]
 profile_info = ("michel", "michel@gmail.com", "12345678")
 marks = {
     "John": 4,
-    "Sergio": 3,
+    "Sergio": 3
 }
 collection_of_coins = {1, 2, 25}
 
-# write your code here
+sorted_variables = {
+    "mutable": [],
+    "immutable": []
+}
+
+all_variables = {}
+all_variables["lucky_number"] = lucky_number
+all_variables["pi"] = pi
+all_variables["one_is_a_prime_number"] = one_is_a_prime_number
+all_variables["name"] = name
+all_variables["my_favourite_films"] = my_favourite_films
+all_variables["profile_info"] = profile_info
+all_variables["marks"] = marks
+all_variables["collection_of_coins"] = collection_of_coins
+
+for values_from_all_variables in all_variables.values():
+    if type(values_from_all_variables) == int \
+            or type(values_from_all_variables) == float \
+            or type(values_from_all_variables) == bool \
+            or type(values_from_all_variables) == str \
+            or type(values_from_all_variables) == tuple:
+        sorted_variables["immutable"].append(values_from_all_variables)
+    else:
+        sorted_variables["mutable"].append(values_from_all_variables)
+
+print(f"{sorted_variables}")
```

Comment:
this solution is correct but too difficult. The task and example say that you can directly write values into the dictionary. Please don't create new variables and functions, too

Reply:
If i didn't create the dict "all_variables", i don't find type of all varables. So, can i use the next directly structure?
`sorted_variables["immutable"].append(lucky_number)` or `sorted_variables["immutable"].append(pi)`  ?

Reply:
Yes, you can. Even more, you can avoid using `append` method. Just write something like this:

```python
dictionary_example = {
   "key_example": [first_value, second_value]
}
```